game/script-jokes.rpy

```diff
@@ -1081,3 +1101,50 @@ label joke_action_figures:
     extend 1fsqflsbr " figures."
 
     return
+
+label joke_befriending_sharks:
+    n 1ccsss "Heh.{w=0.75}{nw}"
+    extend 1fsqss " You ready,{w=0.2} [player]?{w=0.75}{nw}"
+    extend 1fnmbg " You're gonna be like a fish out of water after this one."
+    n 1fcsaj "So!{w=0.75}{nw}"
+    extend 1fcsbg " How do you go about making friends with a shark?"
+    n 1fsqsm "..."
+    n 1fsqss "No?{w=0.75}{nw}"
+    extend 1flrss " Wow,{w=0.2} [player]..."
+    n 1fsgbg "Isn't it obvious?"
+    n 1fllss "You just gotta{w=0.75}{nw}"
+    extend 1fsqbg " {i}chum{/i}{w=0.75}{nw}"
+    extend 1nchgn " them up first!"
+
+    return
+
+label joke_fisherman_broadcast:
+    n 1csqfl "...Why is it so hard to get a video call with a fisherman?"
+    n 1csrca "..."
+    n 1ccsfresi "..."
```

Comment:
I feel like with using the sigh bubble it would be better to use a slightly opened mouth instead of a complete closed one.

Reply:
Sure